trace-sdk/src/main/java/io/bitrise/trace/network/DataSender.java

```diff
@@ -84,7 +88,7 @@
    * @param networkRequest the given NetworkRequest.
    * @return {@code true} if it is valid, {@code false} otherwise.
    */
-  static boolean validateTraceRequest(@Nullable final NetworkRequest networkRequest) {
```

Comment:
refactored this method because it's not just validating trace requests, it validates all network requests.